doc/release-notes.md

```diff
@@ -104,6 +104,11 @@ Low-level changes
 RPC
 ---
 
+- The RPC `addnode` and the `-addnode` bitcoind startup option now
+optionally support adding manual peers that are block-relay-only.
+On these connections, the node doesn't participate in transaction or
+address relay. (#24170)
```

Comment:
Perhaps add "see bitcoind help documentation and RPC help documentation for parameter semantics" or similar

Reply:
Done, in a slightly shortened form.